### DISCUSSION TITLE
Can't access Copilot despite having being accepted into the Technical Preview

### DISCUSSION THREAD

**Discussion by wadefleming-nz** [community/community#13070] [Copilot Conversations] [answered]
I was accepted into the technical preview on 1 Feb 2022.

Today I installed the VS Code extension (windows), it has asked to authenticate against my Github account which was done successfully. 

Back in VS Code it says _'You don't have access to Github Copilot. Join the waitlist....'_

As in another answer I tried signing out of Github using the Settings icon in VS Code. I restarted, logged in and for the same denial message.

**Comment by davecheney** [comment, marked as answer]
> I was accepted into the technical preview on 1 Feb 2022.

@wadefleming-nz looking at the waitlist I do not see an entry for your github username

**Reply by wadefleming-nz** [reply]
Can you see anything there for `wadefleming-sm` or `wflemingnz`?

**Reply by davecheney** [reply]
`wflemingnz` is the one to use.

**Reply by wadefleming-nz** [reply]
thanks

**Comment by nonnorm** [comment]
I’m getting the same problem. I was accepted yesterday, March 22, but I’m still getting the error message.

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by johnathon-walnut** [comment]
I am also getting this issue on regular Visual Studio, no option to Sign out as I can't Sign in and when I Sign in it says in the Output window `You don't have access to GitHub Copilot.`, despite inputting the code it gave me earlier.

**Reply by davecheney** [reply]
Do you have several GitHub accounts?

**Reply by davecheney** [reply]
I can see that [johnathon-walnut](https://github.com/johnathon-walnut) has not been issued an invite. Is it possible you have a second github account registered to your email address?

**Reply by davecheney** [reply]
Also, please see https://github.com/github/feedback/discussions/13070#discussioncomment-2441081 wrt. VSCode vs OSS Code / VSCodium.

**Reply by johnathon-walnut** [reply]
Oops haha, I was logged into my dad's email and thought it was addressed to me...

**Comment by visionm45** [comment]
Same. I am getting
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

after receiving an email yesterday saying
Welcome to GitHub Copilot Technical Preview!

I hope that we can find a solution :)

**Reply by davecheney** [reply]
Do you have several GitHub accounts?

**Reply by visionm45** [reply]
no and actually I just realized that if I go to sign up for the waitlist now it says

GitHub Copilot is already enabled on your account! [Install GitHub Copilot](https://github.com/github/copilot-docs) now.

**Reply by visionm45** [reply]
well actually I successfully signed in now.  seems that it was actually some kind of issue with my access token because after being prompted to log in via the web browser then return to vs code I have access

**Reply by visionm45** [reply]
So to tell the truth I was facing this problem using vscodium.  When I installed the plugin in vscode there was no problem.  So for me at least the culprit seems to be vscodium.

**Reply by davecheney** [reply]
Yup, sorry, only VSCode works.

**Reply by visionm45** [reply]
Thanks

**Comment by aaron-pham** [comment]
Hello, I am having the same issue. I confirmed my email is attached to this github account. This is on IntelliJ IDEA. Can you please help?

**Reply by theandrewgrass** [reply]
Same here! :(

**Reply by Chibuokem** [reply]
I am also having same issue, i have even tried severally to signup and it is refusing to give me access, please what is going on

**Reply by nonnorm** [reply]
I got an email about it:
![Screen Shot 2022-06-21 at 16 36 10](https://user-images.githubusercontent.com/50060110/174901350-5f047b45-b9c2-4960-ba7b-a2493047c714.png)

**Reply by AndriusFromLTU** [reply]
I also have the same issue

**Comment by JuanDavidBallesteros** [comment]
Hi @davecheney,

I have a similar issue, but I can't access Copilot. When I try to log in redirects me to the billing account. I already activated the Copilot in my account.

The output is this:
![image](https://user-images.githubusercontent.com/71021192/189141084-396c1853-4054-49dd-ad45-bdac007dbb02.png)

Finally, I didn't find any solution. Can you help me? Also, I checked to log in with the correct account.

Thank you!

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by Beyondo** [comment]
Just started to happen right now to me. I'm approved since Sep 1st, but now suddenly Copilot stopped working.

**Reply by Akash1134** [reply]
👋 Hi @Beyondo , Thank you for taking the time to report this issue.

Apologies for that temporary issue on our end. Please retry logging in to your IDE plugin, and Copilot should work as expected. 

If you run into any further issues, please reply back in detail with any useful screenshots.

**Comment by ghost** [comment]
hi, I have the exact same issue:
[ERROR] [default] [2023-05-18T12:27:28.103Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as RubenHagege."
[INFO] [auth] [2023-05-18T12:52:37.553Z] Invalid copilot token: missing token: 403 
I did receive a mail, but maybe it's somehow linked to another username that I'm not aware of?
